samples/js/chat_sample/README.md

```diff
@@ -0,0 +1,49 @@
+# JavaScript chat_sample that supports most popular models like LLaMA 3
+
+This example showcases inference of text-generation Large Language Models (LLMs): `chatglm`, `LLaMA`, `Qwen` and other models with the same signature. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample fearures `Pipeline.LLMPipeline` and configures it for the chat scenario.
+
+## Download and convert the model and tokenizers
+
+To convert model you have to use python package `optimum-intel`.
+The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
+
+Install [../../export-requirements.txt](../../export-requirements.txt) to convert a model.
+
+```sh
+pip install --upgrade-strategy eager -r ../../export-requirements.txt
+optimum-cli export openvino --trust-remote-code --model TinyLlama/TinyLlama-1.1B-Chat-v1.0 TinyLlama-1.1B-Chat-v1.0
+```
+
+## Run:
+
+Create `bin` folder in [../../../src/js](../../../src/js).
+Compile GenAI JavaScript bindings archive first. Put its content into `bin` directory.
```

Comment:
The sample should assume that bingings are already prepared. You should reference the corresponding readme instead

Reply:
Done